control/autoware_control_validator/include/autoware/control_validator/control_validator.hpp

```diff
@@ -86,6 +87,9 @@ class ControlValidator : public rclcpp::Node
   void calc_velocity_deviation_status(
     const Trajectory & reference_trajectory, const Odometry & kinematics);
 
+  void calc_stop_point_overrun_status(
```

Comment:
Please consider adding Doxygen-style comment as following

```suggestion
  /**
   * @brief Calculate whether the vehicle has overrun a stop point in the trajectory.
   * @param reference_trajectory Reference trajectory
   * @param kinematics Current vehicle odometry including position and velocity
   * @details Determines the distance to the nearest stop point and checks if the vehicle has
   *          passed beyond the stop point while still moving.
   */
  void calc_stop_point_overrun_status(
```